app/controllers/all/opening/projects_controller.rb

```diff
@@ -13,6 +13,19 @@ def revised
     @date = "#{Date::MONTHNAMES[month.to_i]} #{year}"
   end
 
+  def download_csv
+    authorize Project, :index?
+
+    @projects = ProjectsFetcher.new.sorted_openers(month, year)
+    @date = "#{Date::MONTHNAMES[month.to_i]} #{year}"
+    @month = month
+    @year = year
+
+    @csv = OpeningProjectsCsvExporter.new(@projects).call
+
+    send_data @csv, filename: "opener.csv", type: :csv, disposition: "attachment"
```

Comment:
I'd change the filename to have the month & year in it, so that the user can download multiple CSVs and not have to manually amend the filename.